stdlib/GMP_jll/test/runtests.jl

```diff
@@ -3,6 +3,6 @@
 using Test, Libdl, GMP_jll
 
 @testset "GMP_jll" begin
-    vn = VersionNumber(unsafe_string(unsafe_load(cglobal((:__gmp_version, libgmp), Ptr{Cchar}))))
+    vn = VersionNumber(unsafe_string(unsafe_load(cglobal(dlsym(libgmp, :__gmp_version), Ptr{Cchar}))))
```

Comment:
Should/could cglobal be taught how to handle a LazyLibrary?

Reply:
It could, the most annoying thing is just that `cglobal()` is an intrinsic, not a function, so to add an overload for that involves writing C code, not just Julia code.